backends/gstreamer/lib.rs

```diff
@@ -346,11 +346,11 @@ pub fn set_element_flags<T: glib::IsA<gst::Object> + glib::IsA<gst::Element>>(
     flags: gst::ElementFlags,
 ) {
```

Comment:
That's obsoleted by https://gstreamer.pages.freedesktop.org/gstreamer-rs/stable/latest/docs/gstreamer/prelude/trait.ElementExtManual.html#tymethod.set_element_flags I think

Reply:
Yep :)